src/components/space-usage/SpaceUsage.js

```diff
@@ -3,7 +3,7 @@ import Chart from "assets/img/illustration/chart.svg";
 
 const SpaceUsage = () => {
   return (
-    <div className="flex flex-wrap items-end content-end justify-center mt-6 relative h-32">
+    <div className="flex flex-wrap items-end content-end justify-center mt-6 relative h-40">
```

Comment:
You don't need to change the height. It will break the UI when the screen resolution is lower than 1920 x 1080.
```suggestion
    <div className="flex flex-wrap items-end content-end justify-center mt-6 relative h-32">
```